### DISCUSSION TITLE
option for default file using fugitive_gI

### DISCUSSION THREAD

**Discussion by ANaumann85** [tpope/vim-fugitive#2165] [Ideas]
The mapping  `gi` opens the file `.git/info/exclude`, whereas `1gi` opens `.gitignore`  (relative to the repositories root).

Is it possible to switch that behavior with an option, so that `gi` opens `.gitignore` and the counter opens `.git/info/exlude`?

**Comment by tpope** [comment]
No option. I would expect it to be possible with a custom map, but you're on your own to implement that.

The reason for the current design is two-fold:

1. `gI` is quick-and-dirty, which is a good fit for `.git/info/exclude`. `.gitignore` tends to be more organized, so throwing a new ignore on the last line is less useful.
2. The full intended design was `1gI` for `.gitignore`, `2gI` for `dir/.gitignore`, `3gI` for `dir/subdir/.gitingore`, etc. This isn't currently implemented, but that's a reason the count is on the `.gitginore` side.
